.github/workflows/code-quality.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
           if [[ -z "$clean" ]]; then
               echo "Empty git status --porcelain: $clean"
           else
+              echo "::error::Please run the command from the previous step (Rerun all code generators...) and commit the result."
```

Comment:
Please, instead of telling to run command from previous step, maybe we should explicitly say what to run.

Reply:
Ok, then, let me change this.

Reply:
done